.github/workflows/image-release.yml

```diff
@@ -18,10 +18,6 @@ jobs:
     steps:
       - uses: actions/checkout@v4
       - uses: ./.github/actions/setup
-      - name: set version
```

Comment:
Looks unrelated to me. Is this change intended?

Reply:
This is actually related to https://github.com/gardenlinux/glrd/pull/3/files#diff-50c86b7ed8ac2cf95bd48334961bf0530cdc77b5a56f852c5c61b89d735fd711R25 and the switch to poetry. I want to try out dynamic versioning (based on git tag).
Not sure if this will work though. Might need some adaptions after the merge before a new version can be released.